deltachat-ffi/deltachat.h

```diff
@@ -523,6 +523,7 @@ char*           dc_get_blobdir               (const dc_context_t* context);
  *                    e.g. `ui.desktop.foo`, `ui.desktop.linux.bar`, `ui.android.foo`, `ui.dc40.bar`, `ui.bot.simplebot.baz`.
  *                    These keys go to backups and allow easy per-account settings when using @ref dc_accounts_t,
  *                    however, are not handled by the core otherwise.
+ *- `enable_realtime` = Whether the realtime APIs should be enabled. If not enabled, they will behave as noops.
```

Comment:
isn't that problematic for the webxdcs, if they can't know if the apis are even usable?